runtimes/datascience/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -330,6 +330,9 @@ fi
 COPY ${DATASCIENCE_SOURCE_CODE}/pylock.toml ./
 # Copy Elyra dependencies for air-gapped enviroment
 COPY ${DATASCIENCE_SOURCE_CODE}/utils ./utils/
+# Download Elyra boostrapper.py
+RUN curl -fL https://raw.githubusercontent.com/opendatahub-io/elyra/refs/heads/main/elyra/kfp/bootstrapper.py \
```

Comment:
This will not work in disconnected envs.

Reply:
Hello @atheo89 I thought that it would work because this would be done during the image build time. The idea is now that Notebook images don't need a custom `bootstrapper.py` we could download it from Elyra and use the new system property we introduced: `ELYRA_INSTALL_PACKAGES` to prevent Elyra from trying to download packages - this new flag should do that